src/concepts/ecosystem.md

```diff
@@ -0,0 +1,24 @@
+# DataJoint Ecosystem 
+
+An overview of the DataJoint ecosystem is provided below: 
+
+### Open Source softwares
+
++ DataJoint Python - DataJoint APIs in Python: [https://github.com/datajoint/datajoint-python](https://github.com/datajoint/datajoint-python)
++ DataJoint MATLAB - DataJoint APIs in MATLAB: [https://github.com/datajoint/datajoint-matlab](https://github.com/datajoint/datajoint-matlab)
++ DataJoint LabBook - data entry GUI: [https://github.com/datajoint/datajoint-labbook](https://github.com/datajoint/datajoint-labbook)
++ DataJoint CodeBook - Jupyter-based coding environment: [https://codebook.datajoint.io](https://codebook.datajoint.io)
++ DataJoint SciViz - data visualization framework: [https://github.com/datajoint/sci-viz](https://github.com/datajoint/sci-viz)
++ DataJoint Docker image for mysql: [https://github.com/datajoint/mysql-docker](https://github.com/datajoint/mysql-docker)
++ DataJoint Pharus - generic REST API server backend: [https://github.com/datajoint/pharus](https://github.com/datajoint/pharus)
```

Comment:
Since these are already on the [Core](https://datajoint.com/docs/core/) page, we can remove from here.  CodeBook will be added as a separate page in the future.